src/components/ReportActionItem/RenameAction.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Text from '@components/Text';
+import withCurrentUserPersonalDetails, {type WithCurrentUserPersonalDetailsProps} from '@components/withCurrentUserPersonalDetails';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import CONST from '@src/CONST';
+import type {ReportAction} from '@src/types/onyx';
+
+type RenameActionProps = WithCurrentUserPersonalDetailsProps & {
+    /** All the data of the action */
+    action: ReportAction;
+};
+
+function RenameAction({currentUserPersonalDetails, action}: RenameActionProps) {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const currentUserAccountID = currentUserPersonalDetails.accountID ?? '';
+    const userDisplayName = action.person?.[0]?.text;
+    const actorAccountID = action.actorAccountID ?? '';
+    const displayName = actorAccountID === currentUserAccountID ? `${translate('common.you')}` : `${userDisplayName}`;
+    const originalMessage = action.actionName === CONST.REPORT.ACTIONS.TYPE.RENAMED ? action.originalMessage : null;
```

Comment:
NAB: Why only access it if it's a renamed action? Don't we already know this is a rename action?

Reply:
@neil-marcellini It's more for the TypeScript to narrow down the type and know what fields to expect in `originalMessage` property